app/controllers/application_controller.rb

```diff
@@ -83,4 +83,15 @@ def set_sentry_user
 
     Sentry.set_user(id: current_user.id)
   end
+
+  def track_validation_error(form)
+    ValidationError.create!(
+      form_object: form.class.name,
+      request_path: request.path,
+      user: current_user,
+      details: form.errors.messages.to_h { |field, messages| [field, { messages:, value: (form.public_send(field) if form.respond_to?(field)) }] },
```

Comment:
Probably not a huge consideration but is there any risk of this containing PII?

Reply:
Yes @Lockyy. It is supposed to show emails or any other info put into forms.